### ISSUE TITLE
Lint isn't applied to interface members.

### ISSUE THREAD

**Issue by wildfunctions** [ajafff/tslint-consistent-codestyle#136]
I noticed the linter was not catching errors with my interface's members, so I did a quick test with the following examples:

```
{ "type": "member", "format": "camelCase" }
```

Will correctly error with the class example below:

```
export class FooBar {
    foo_bar: string
}
```

But will not error with the interface example below:

```
export interface IFooBar {
    foo_bar: string
}
```
